src/main/java/org/ecocean/Decision.java

```diff
@@ -63,6 +69,104 @@ public long getTimestamp() {
         return timestamp;
     }
 
+    public static void updateEncounterStateBasedOnDecision(Shepherd myShepherd, Encounter enc){
+      String context="context0";
+      List<Decision> decisionsForEncounter = myShepherd.getDecisionsForEncounter(enc);
+      if(decisionsForEncounter != null && decisionsForEncounter.size() > 0){
+        int MIN_DECISIONS_TO_CHANGE_ENC_STATE = (new Integer(CommonConfiguration.getProperty("MIN_DECISIONS_TO_CHANGE_ENC_STATE",context))).intValue();
+        int numberOfMatchDecisionsMadeForEncounter = Decision.getNumberOfMatchDecisionsMadeForEncounter(decisionsForEncounter);
+        System.out.println("Decision numberOfMatchDecisionsMadeForEncounter is: " + numberOfMatchDecisionsMadeForEncounter);
+        if(getNumberOfMatchDecisionsMadeForEncounter(decisionsForEncounter) >= MIN_DECISIONS_TO_CHANGE_ENC_STATE){
+          System.out.println("Decision " + getNumberOfMatchDecisionsMadeForEncounter(decisionsForEncounter) + " decisions have been made about the ecounter, which is at or above the " + MIN_DECISIONS_TO_CHANGE_ENC_STATE + " count threshold.");
+          int numberOfAgreementsForMostAgreedUponMatch = Decision.getNumberOfAgreementsForMostAgreedUponMatch(decisionsForEncounter);
+          int MIN_AGREEMENTS_TO_CHANGE_ENC_STATE = (new Integer(CommonConfiguration.getProperty("MIN_AGREEMENTS_TO_CHANGE_ENC_STATE",context))).intValue();
+          if(numberOfAgreementsForMostAgreedUponMatch >= MIN_AGREEMENTS_TO_CHANGE_ENC_STATE){
+            System.out.println("Decision numberOfAgreementsForMostAgreedUponMatch is: " + numberOfAgreementsForMostAgreedUponMatch + ", which is at or above the "+ MIN_AGREEMENTS_TO_CHANGE_ENC_STATE + " count threshold");
+            try{
+              String newState = "mergereview";
+              enc.setState(newState);
+              myShepherd.updateDBTransaction();
+            }catch(Exception e){
+              System.out.println("Error trying to update encounter state in Decision.updateEncounterStateBasedOnDecision()");
+              e.printStackTrace();
+            }
+            finally{
+              myShepherd.rollbackDBTransaction();
+            }
+          }else{
+            if(numberOfAgreementsForMostAgreedUponMatch < MIN_AGREEMENTS_TO_CHANGE_ENC_STATE){
```

Comment:
i think this `if() {` block may be unnecessary, if i am keeping track of all my nestings correctly!  i believe the `else` right above it is tied to `if(numberOfAgreementsForMostAgreedUponMatch >= MIN_AGREEMENTS_TO_CHANGE_ENC_STATE)` above... which means (i think!) this `if` is the only case we could be in here.... does that make sense?   i dont think this **affect functionality** but may just be adding an unnecessary complexity.

Reply:
Agreed completely. Just wanted to be super explicit about the case in which I wanted the disputed state to happen for readability.